### PR TITLE
gulp-task-to-create-jwt

### PR DESCRIPTION
**Pull request by nehaabrol87**
@morrissinger @jonathanbarton 👀

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -208,6 +208,14 @@ gulp.task('default', ['clean'], () => {
   );
 });
 
+//Generates new JWT
+gulp.task('createNewToken',[],() => {
+  let jwtSecret = config.jwtSecret;
+  let jwt = require('jsonwebtoken');
```
Comment (morrissinger) [inline gulpfile.babel.js]: A `const` at the top-level scope may be more appropriate.

File: gulpfile.babel.js [hunk 2]
```diff
@@ -208,6 +208,14 @@ gulp.task('default', ['clean'], () => {
...
+  let token = jwt.sign({data : {name: 'foo',password: 'bar'}},jwtSecret , { expiresIn: '1h' });
```
Comment (morrissinger) [inline gulpfile.babel.js]: Can we set foo/bar in constants in another file, from the environment, or from the command line?

File: gulpfile.babel.js [hunk 3]
```diff
@@ -208,6 +208,14 @@ gulp.task('default', ['clean'], () => {
...
+  console.log(`GENERATED TOKEN ${token}`);
```
Comment (morrissinger) [inline gulpfile.babel.js]: Can we separate 'GENERATED TOKEN' from the token with a colon or a new line?

**Review by morrissinger — CHANGES_REQUESTED** [review]

File: gulpfile.babel.js (hunk 1 above)
Comment (nehaabrol87) [inline gulpfile.babel.js]: yes

File: gulpfile.babel.js (hunk 3 above)
Comment (nehaabrol87) [inline gulpfile.babel.js]: yes sounds good.

File: gulpfile.babel.js (hunk 2 above)
Comment (nehaabrol87) [inline gulpfile.babel.js]: Not sure how can these be fetched from the environment.
Are you suggesting adding below object in another file?
data : {
  name : 'foo',
 password : 'bar'
}

**Pull request closed without merging**